backend/src/test/resources/ID016_modifyIdea.feature

```diff
@@ -45,18 +45,18 @@ Feature: Modify Idea
     Then the idea with id "<id>" will have value "<new_value>" for the field "<field>"
 
     Examples:
-      | id | field       | new_value      |
-      | 18 | title       | Health App     |
-      | 18 | purpose     | For customer   |
-      | 19 | description | new extra info |
-      | 18 | isPaid      | True           |
-      | 18 | inProgress  | True           |
-      | 19 | isPrivate   | True           |
-      | 18 | domains     | 3              |
-      | 19 | topics      | 7              |
-      | 18 | techs       | 8              |
-      | 19 | image URLs  | 13             |
-      | 18 | icon URL    | 13             |
+      | id | field       | new_value                   |
+      | 18 | title       | Health App                  |
+      | 18 | purpose     | For customer                |
+      | 19 | description | new extra info              |
+      | 18 | isPaid      | True                        |
+      | 18 | inProgress  | True                        |
+      | 19 | isPrivate   | True                        |
+      | 18 | domains     | 3                           |
+      | 19 | topics      | 7                           |
+      | 18 | techs       | 8                           |
+      | 19 | image URLs  | www.test.com, www.test2.com |
```

Comment:
There is an extra space here in front of www.test2.com

Reply:
Thanks!